docstrfmt/docstrfmt.py

```diff
@@ -837,7 +837,14 @@ def literal(
     def literal_block(
         self, node: docutils.nodes.literal_block, context: FormatContext
     ) -> line_iterator:
-        yield "::"
+        if (
+            len(node.attributes["classes"]) > 1
+            and node.attributes["classes"][0] == "code"
+        ):
+            args = "".join([f" {arg}" for arg in node.attributes["classes"][1:]])
+            yield f".. code::{args}"
```

Comment:
This will need to be `code-block`:

```suggestion
            yield f".. code-block::{args}"
```